core/services/ocr2/plugins/ocr2keeper/evmregistry/v21/upkeepstate/store.go

```diff
@@ -320,7 +320,9 @@ func (u *upkeepStateStore) cleanup(ctx context.Context) error {
 func (u *upkeepStateStore) cleanDB(ctx context.Context) error {
 	tm := time.Now().Add(-1 * u.retention)
 
-	return u.orm.DeleteExpired(tm, pg.WithParentCtx(ctx), pg.WithLongQueryTimeout())
+	ctx, cancel := context.WithTimeout(sqlutil.WithoutDefaultTimeout(ctx), time.Minute)
```

Comment:
Should we remove the timeout completely?

Reply:
How come? I think these are still necessary to replace the default timeout which may be too short.

Reply:
Just wondering if it's ok to never timeout, compared to what we have now. I don't have a strong preference, just noticed the difference.

Reply:
This change is preserving the original "long" timeout of `1m`.